tls-tester/pom.xml

```diff
@@ -15,11 +15,10 @@
       <plugin>
         <groupId>com.axonivy.ivy.ci</groupId>
         <artifactId>project-build-plugin</artifactId>
-        <version>8.0.6</version>
+        <version>10.0.14</version>
```

Comment:
the CI build is not running. so you should first address that problem.
I think all you need is to change the java version to 17 .. maybe also the provider.
https://github.com/axonivy/tls-connection-tester/blob/master/.github/workflows/maven.yml#L25